### PR TITLE
Next.js (Vercel) topic revisions.

### PR DESCRIPTION
**Pull request by danieltprice**
- Current Next.js (Vercel) topic revisions, We'll revisit this content for the Vercel integration.
- Move under Develop heading in TOC
https://websitemain62807-dpriceverceltopicrevisions.gtsb.io/docs/integrations/vercel/

### CODE REVIEW

File: content/docs/integrations/vercel.md [hunk 1]
```diff
@@ -1,43 +1,63 @@
...
+
+Add your Neon connection details to your `.env` file.
 
 ```shell
-NEON_HOST=...
-NEON_DB=...
-NEON_USER=...
-NEON_PASS=...
-NEON_PORT=...
+NEON_HOST='<project_id>.cloud.neon.tech'
```
Comment (duskpoet) [inline content/docs/integrations/vercel.md:43]: I think connection string method is rather easier to use

Comment (danieltprice) [reply content/docs/integrations/vercel.md:43]: Hi @duskpoet 
Happy to use the connection string instead. In this case, do I simply drop the connection string with the password included in the .env file as provided on the Dashboard?

And in that case, how should I modify the connection snippet that we provide?

import postgres from 'postgres';

const sql = postgres({
  host: process.env.NEON_HOST,
  port: process.env.NEON_PORT,
  database: process.env.NEON_DB,
  username: process.env.NEON_USER,
  password: process.env.NEON_PASS,
});

const result = await sql.uafe(req.body);

Comment (duskpoet) [reply content/docs/integrations/vercel.md:43]: .env
```
DATABASE_URL=<neon_connection_string>
```
index.ts
```
const sql = postgres(process.env.DATABASE_URL);
```

Comment (danieltprice) [reply content/docs/integrations/vercel.md]: Thank you!

File: content/docs/integrations/vercel.md [hunk 2]
```diff
@@ -1,56 +1,67 @@
...
-const sql = postgres({
-  host: process.env.NEON_HOST,
-  port: process.env.NEON_PORT,
-  database: process.env.NEON_DB,
-  username: process.env.NEON_USER,
-  password: process.env.NEON_PASS,
-});
+const sql = postgres(process.env.DATABASE_URL);
 
 const result = await sql.uafe(req.body);
```
Comment (duskpoet) [inline content/docs/integrations/vercel.md:64]:
```suggestion
const result = await sql.unsafe(req.body);
```

**Review by duskpoet — APPROVED** [review]

**Review by duskpoet — APPROVED** [review]

**Pull request merged**